be/src/olap/field.h

```diff
@@ -57,6 +57,7 @@ class Field {
 
     inline void set_to_max(char* buf) const { return _type_info->set_to_max(buf); }
     inline void set_to_min(char* buf) const { return _type_info->set_to_min(buf); }
+    inline char* get_type_value_with_arena(Arena* arena) const { return _type_info->get_type_value_with_arena(arena); }
```

Comment:
It's not necessary to copy all methods of TypeInfo to Field, I'd prefer to remove `get_type_value_with_arena` and `direct_copy_content` from this class.

Reply:
Field defines column's content info,TypeInfo defines all type column's behavior;
In column_zone_map_builder,caller just need to care column abstract behavior,but not to known its realization.Different type has different realization.
Such as,when calling get_type_value_with_arena,int/double type not need data length,while string type need data length,so int/double type and char type has different method.
I think using Field's to realize polymorphism is better